monailabel/scribbles/utils.py

```diff
@@ -12,6 +12,9 @@
 
 import numpy as np
 import numpymaxflow
+import torch
+from monai.networks.layers import GaussianMixtureModel
+from sklearn.mixture import GaussianMixture
```

Comment:
It would be better to avoid an additional dependency (sklearn).

Reply:
Agreed! Perhaps for now we can remove this dependency and raise an error when tensors are not on GPU (we still have the other histogram-based method that should be usable on CPU).